src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,27 @@
+create table conversation (
+    id bigint not null,
+    created_date datetime not null,
+    updated_date datetime not null,
+    conversation_time datetime,
+    message longblob,
```

Comment:
```suggestion
    id bigint AUTO_INCREMENT not null,
    created_date datetime not null,
    updated_date datetime not null,
    conversation_time datetime,
    message longtext,
```
일때 local 에서 정상작동 되던데 혹시 CU는 로컬에서 잘 작동 하셨었나요?

Reply:
별도의 설정을 해주지 않아도 됩니다. gradle reimport 한번해보시겠어요?

message longblob 타입을 설정해둔 이유는 이미지 파일을 염두해서 넣어두었던 것인데요. 
흠.. 이미지나 파일을 추가할 경우에 현재는 슬랙에서 링크를 추가하는데요. 
유효기간이 얼마나 될지 모르니, S3로 옮기고 해당 링크를 바라보도록 기능을 추가해야겠네요


```json
{
            "type": "message",
            "text": "",
            "files": [
                {
                    "id": "F0144J6F7EK",
                    "created": 1590912507,
                    "timestamp": 1590912507,
                    "name": "final.png",
                    "title": "final.png",
                    "mimetype": "image\/png",
                    "filetype": "png",
                    "pretty_type": "PNG",
                    "user": "UJ71JS7K7",
                    "editable": false,
                    "size": 118821,
                    "mode": "hosted",
                    "is_external": false,
                    "external_type": "",
                    "is_public": true,
                    "public_url_shared": false,
                    "display_as_bot": false,
                    "username": "",
                    "url_private": "https:\/\/files.slack.com\/files-pri\/TFELTJB7V-F0144J6F7EK\/final.png",
                    "url_private_download": "https:\/\/files.slack.com\/files-pri\/TFELTJB7V-F0144J6F7EK\/download\/final.png",
                    "thumb_64": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_64.png",
                    "thumb_80": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_80.png",
                    "thumb_360": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_360.png",
                    "thumb_360_w": 360,
                    "thumb_360_h": 265,
                    "thumb_480": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_480.png",
                    "thumb_480_w": 480,
                    "thumb_480_h": 353,
                    "thumb_160": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_160.png",
                    "thumb_720": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_720.png",
                    "thumb_720_w": 720,
                    "thumb_720_h": 530,
                    "thumb_800": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_800.png",
                    "thumb_800_w": 800,
                    "thumb_800_h": 589,
                    "thumb_960": "https:\/\/files.slack.com\/files-tmb\/TFELTJB7V-F0144J6F7EK-5c1633c37c\/final_960.png",
                    "thumb_960_w": 960,
                    "thumb_960_h": 706,
                    "original_w": 980,
                    "original_h": 721,
                    "thumb_tiny": "AwAjADCuelNJp+fl7UwnpTEGaOaKKQw5o7daSigB\/wDDTTTv4aaaYBx70lLmkpAFFGaXNABRQaKACiiigBR0NAHWgdDSjvTQj\/\/Z",
                    "permalink": "https:\/\/woowacourse.slack.com\/files\/UJ71JS7K7\/F0144J6F7EK\/final.png",
                    "permalink_public": "https:\/\/slack-files.com\/TFELTJB7V-F0144J6F7EK-5d366c211a",
                    "is_starred": false,
                    "has_rich_preview": false
                }
            ],
            "upload": true,
            "user": "UJ71JS7K7",
            "display_as_bot": false,
            "ts": "1590912509.003800",
            "client_msg_id": "9ebbd910-5881-421e-bab7-dd7428f0f4ae",
            "thread_ts": "1590911956.002900",
            "parent_user_id": "USSE91Z7X"
        },
```